cmd/chartmuseum/main.go

```diff
@@ -107,6 +109,18 @@ func cliHandler(c *cli.Context) {
 		crash(err)
 	}
 
+	go func() {
+		t := time.NewTicker(time.Minute * 5)
+		for _ = range t.C {
+			server.(*multitenant.MultiTenantServer).RebuildIndex()
+		}
+	}()
+
+	go func() {
+		multitenant.InitUpdateObjectChan()
+		server.(*multitenant.MultiTenantServer).Consumer()
+	}()
+
```

Comment:
It appears that this modifies the default behavior, and that `RebuildIndex` will be run every 5 minutes on a timer (vs. on demand). Is that correct? 

I think that this should only occur based on the presence of some new flag, such as `--cache-interval=5m`.

These goroutines should also be encapsulated in some method on the server, such as `InitCacheTimer`

Reply:
Maybe the name `RebuildIndex`  is not
 very well, it delta update the cache every 5min (in case the files on the disk are manually manipulated).

It is a good idea to set a new flag and encapsulate these goroutines.